modules/virt-define-http-liveness-probe.adoc

```diff
@@ -4,74 +4,36 @@
 
 [id="virt-define-http-liveness-probe_{context}"]
 
-= Define an HTTP liveness probe
+= Defining an HTTP liveness probe
+
+Define an HTTP liveness probe by setting the `spec.livenessProbe.httpGet` field of the virtual machine instance (VMI) configuration.
 
-This procedure provides an example configuration file for defining HTTP
-liveness probes.
 
 .Procedure
 
-. Customize a YAML configuration file to create an HTTP liveness probe, using
-the following code block as an example. In this example:
-+
---
-* You configure a probe using `spec.livenessProbe.httpGet`, which queries port `1500` of the virtual machine instance, after an initial delay of `120` seconds.
-* The virtual machine instance installs and runs a minimal HTTP server
-on port `1500` using `cloud-init`.
---
-+
-[NOTE]
-====
-The `timeoutSeconds` value must be lower than the `periodSeconds` value. The `timeoutSeconds` default value is `1`. The `periodSeconds` default value is `10`.
-====
+. Edit the VMI configuration file to include details of the HTTP liveness probe.
 +
+
+.Sample liveness probe with an HTTP GET test 
 [source,yaml]
 ----
-apiVersion: kubevirt.io/v1alpha3
-kind: VirtualMachineInstance
-metadata:
-  labels:
-    special: vmi-fedora
-  name: vmi-fedora
 spec:
-  domain:
-    devices:
-      disks:
-      - disk:
-          bus: virtio
-        name: containerdisk
-      - disk:
-          bus: virtio
-        name: cloudinitdisk
-    resources:
-      requests:
-        memory: 1024M
   livenessProbe:
-    initialDelaySeconds: 120
-    periodSeconds: 20
-    httpGet:
-      port: 1500
-    timeoutSeconds: 10
-  terminationGracePeriodSeconds: 0
-  volumes:
-  - name: containerdisk
-    registryDisk:
-      image: kubevirt/fedora-cloud-registry-disk-demo
-  - cloudInitNoCloud:
-      userData: |-
-        #cloud-config
-        password: fedora
-        chpasswd: { expire: False }
-        bootcmd:
-          - setenforce 0
-          - dnf install -y nmap-ncat
-          - systemd-run --unit=httpserver nc -klp 1500 -e '/usr/bin/echo -e HTTP/1.1 200 OK\\n\\nHello World!'
-    name: cloudinitdisk
+    initialDelaySeconds: 120 <1>
+    periodSeconds: 20 <2>
+    httpGet: <3>
+      port: 1500 <4>
```

Comment:
I think we should add more details on the probe, an example from the official K8s docs:
```yaml
  livenessProbe:
      httpGet:
        path: /healthz
        port: 8080
        httpHeaders:
        - name: Custom-Header
          value: Awesome
      initialDelaySeconds: 3
      periodSeconds: 3
```
I'd at least add path as well.

Reply:
Added